executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model/ProjectView.kt

```diff
@@ -3,72 +3,70 @@ package org.jetbrains.bsp.bazel.projectview.model
 import io.vavr.collection.Seq
 import io.vavr.control.Try
 import org.apache.logging.log4j.LogManager
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewBazelPathSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewBuildFlagsSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewDebuggerAddressSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewExcludableListSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewJavaPathSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewListSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewSingletonSection
-import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewTargetsSection
+import org.jetbrains.bsp.bazel.projectview.model.sections.*
 
 /**
  * Representation of the project view file.
  *
  * @link https://ij.bazel.build/docs/project-views.html
  */
 data class ProjectView constructor(
-    /** targets included and excluded from the project  */
-    val targets: ProjectViewTargetsSection?,
-    /** bazel path used to invoke bazel from the code  */
-    val bazelPath: ProjectViewBazelPathSection?,
-    /** debugger address which can be added to the server run command (as a flag to java)  */
-    val debuggerAddress: ProjectViewDebuggerAddressSection?,
-    /** path to java to run a server  */
-    val javaPath: ProjectViewJavaPathSection?,
-    /** bazel flags added to all bazel command invocations  */
-    val buildFlags: ProjectViewBuildFlagsSection?,
+        /** targets included and excluded from the project  */
+        val targets: ProjectViewTargetsSection?,
+        /** bazel path used to invoke bazel from the code  */
+        val bazelPath: ProjectViewBazelPathSection?,
+        /** debugger address which can be added to the server run command (as a flag to java)  */
+        val debuggerAddress: ProjectViewDebuggerAddressSection?,
+        /** path to java to run a server  */
+        val javaPath: ProjectViewJavaPathSection?,
+        /** bazel flags added to all bazel command invocations  */
+        val buildFlags: ProjectViewBuildFlagsSection?,
+        /** build manual targets to build, run and test by explicitly asking. */
```

Comment:
this flag will be used only for building, for testing / running we will create another one probably.

so it could be `flag for building manual targets`